pkgs/tools/text/sgrep/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, runCommand, lib, m4, makeWrapper }:
+let
+  sgrep = stdenv.mkDerivation rec {
+    pname = "sgrep";
+    version = "1.94a";
+
+    src = fetchurl {
+      url =
+        "https://www.cs.helsinki.fi/pub/Software/Local/Sgrep/sgrep-${version}.tar.gz";
+      sha256 = "sha256-1bFkeOOrRHNeJCg9LYldLJyAE5yVIo3zvbKsRGOV+vk=";
+    };
+
+    buildInputs = [ makeWrapper ];
```

```suggestion
    nativeBuildInputs = [ makeWrapper ];
```

`makeWrapper` should be in `nativeBuildInputs`

---

Yikes, applied.